rules.mk

```diff
@@ -235,7 +235,7 @@ else
   TARGET_NM:=$(TARGET_CROSS)nm
 endif
 
-BUILD_KEY=$(TOPDIR)/key-build
+BUILD_KEY?=$(TOPDIR)/key-build
```

Comment:
This shouldn't be changed if possible as it will make us different from upstream. Just symlink the keys to this default name instead...